meta-ros-common/classes/ros_cmake.bbclass

```diff
@@ -13,3 +13,8 @@ EXTRA_OECMAKE:prepend:class-native = "\
     -DCMAKE_PREFIX_PATH='${STAGING_DIR_NATIVE}${ros_prefix};${STAGING_DIR_NATIVE}${prefix}' \
     -DCMAKE_INSTALL_PREFIX:PATH='${ros_prefix}' \
 "
+
+EXTRA_OECMAKE:prepend:class-nativesdk = "\
+    -DCMAKE_PREFIX_PATH='${STAGING_DIR_NATIVE}/opt/ros/${ROS_DISTRO};${STAGING_DIR_NATIVE}${ros_prefix};${STAGING_DIR_NATIVE}${prefix}' \
```

Comment:
${STAGING_DIR_NATIVE}/opt/ros/${ROS_DISTRO} is identical to ${STAGING_DIR_NATIVE}${ros_prefix}

Reply:
I will remove and test.

Reply:
Fixed https://github.com/ros/meta-ros/pull/1241/commits/b99fe2b6e538bce8e1c49485c9a75032f870b255